bench/workgen/runner/prepare_stress.py

```diff
@@ -0,0 +1,206 @@
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+
+# This benchmark is designed to stress disk access to the history store file.
+# This is achieved through:
+#   - Long running transactions consisting of read and update operations.
+#   - Low cache size (~20%) for a reasonably sized WT table with large documents.
+#   - Pareto distribution for operations in long running transactions. This will cause
+#     skewed access for a selective set of keys in WT table.
+#   - Relatively large number of read operation threads to stress cache.
+
+# Benchmark is based on a wtperf config: wiredtiger/bench/wtperf/runners/evict-btree-scan.wtperf
+# There are number of changes made to original configs such as:
+#   - Reduced the number of documents inserts during initial warm-up phase.
+#   - Increased the sizes of key and value.
+#   - Reduced the WT cache size.
+#   - Added transaction based operations and repurposed some of the read threads as long-running
+#     transcation threads.
+
+# This file is derived from evict-btree-hs.py and below are the new configurations added
+# - Timestmap support for prepare transactions
+# - Commit the transaction with prepare, commit and durable timestamp (use_prepare_timestamp).
+# - Commit the transaction with commit_timestamp (use_commit_timestamp).
+# - Read timestamp lag from the current time (read_timestamp_lag)
+# - Number of seconds lag to the oldest_timestamp from current time (oldest_timestamp_lag).
+# - Number of seconds lag to the stable_timestamp from current time (stable_timestamp_lag).
+# - Number of seconds to wait before moving oldest and stable timestamp (timestamp_advance).
+# - Thread.options.session_config - Session configuration.
+
+###################################################################################################
+'''
+# wtperf options file: evict btree configuration
+conn_config="cache_size=40G,checkpoint=(wait=60,log_size=2GB),eviction=(threads_min=12,
+threads_max=12),log=(enabled=true),session_max=600,eviction_target=60,statistics=(fast),
+statistics_log=(wait=1,json)"
+
+# 1B records * (key=12 + value=138) is about 150G total data size
+key_sz=12
+value_sz=138
+log_like_table=true
+table_config="type=file"
+icount=1000000000
+report_interval=5
+run_time=3600
+# Scans every 10 minutes for all the scan specific tables.
+# .4B records * (key=12 + value=138) is about 60G total data size for scan
+# Running on a machine with 64G physical memory, this exhausts both the
+# WT cache and the system cache.
+scan_interval=600
+scan_pct=100
+scan_table_count=20
+scan_icount=400000000
+populate_threads=5
+table_count=100
```

Comment:
@raviprakashgiri29, this was my source of confusion.  `table_count=100` here, but was reset to `tablecount = 1` below.  As a matter of fact many of these constants have been reset.  I think after the `#######` line above, I would put:
```
# These wtperf constants were used to originally generate this python file, which has been since edited.
# The table_count, icount, and other variables have been changed below.
```
So you are right, that my change to moving the `txn()` call I called out in the review, will make little/no difference.  But I think the change is worth making, in case table_count is changed, or this code is copied to yet another workload.